src/components/common/ui/form/with-formik/radio/RadioGroup.stories.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useState } from 'react';
+import { Box, Typography } from '@mui/material';
+import type { Meta, StoryObj } from '@storybook/react';
+import { Form, Formik } from 'formik';
+
+import FormikRadioGroup from '@/components/common/ui/form/with-formik/radio/FormikRadioGroup';
+
+type RadioGroupStoryType = {
+  name: string;
+  options: { label: string; value: string }[];
+};
+
+const RadioGroupStory: React.FC<RadioGroupStoryType> = (
+  args: RadioGroupStoryType,
+) => {
+  const [name, setName] = useState('Name');
```

Comment:
This hook is never used. You can delete it as everything works correctly for this version.